src/databricks/labs/ucx/queries/migration/main/04_1_remaining_tables.sql

```diff
@@ -0,0 +1,4 @@
+/* --title 'List of remaining tables in HMS' --width 6 */
```

Comment:
Share a dashboard screenshot after you are done with your changes

Reply:
Will share after finishing the integration test...